docs/antrea-network-policy.md

```diff
@@ -1102,6 +1107,35 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector
+
+NodeSelector selects certain Nodes which match the label selector. It adds Node IPs to egress rules in `to` field
```

Comment:
I just wonder any other CNIs support similar features? Do they use NodeSelector only for from/to or also for appliedTo, or even enforce policies on the Node traffic with NodeSelector?

Reply:
Both Calico and Cilium use NodeSelector also for 'appliedTo', and enforce policies on the Node traffic, AFAIK.
However, supporting host-firewall would be a bigger feature than this.

Reply:
So, appliedTo is always for Node traffic, not for selecting Pods on the Nodes?

Reply:
You are right. I think so.

Reply:
You are right. I think so.

Reply:
Ok.